execution/executor/src/fuzzing.rs

```diff
@@ -32,10 +32,29 @@ pub fn fuzz(
     let db_reader_writer = DbReaderWriter::new(fake_db);
     let mut executor = Executor::<FakeVM>::new(db_reader_writer);
 
-    //
     let _events = executor.execute_and_commit_chunk(txn_list_with_proof, verified_target_li, None);
 }
 
+pub fn fuzz_execute_and_commit_blocks(
+    blocks: Vec<(HashValue, Vec<Transaction>)>,
+    ledger_info_with_sigs: LedgerInfoWithSignatures,
+) {
+    // setup fake db
```

Comment:
nit: you could probably save some lines by defining a helper `create_test_executor()`, which is called by both fuzzing methods.

Reply:
sg